kroxylicious-operator/install/01.ClusterRole.kroxylicious-operator-watched.yaml

```diff
@@ -28,12 +28,31 @@ rules:
       # - create
       # - patch
       # - update
+  - apiGroups:
+      - "filter.kroxylicious.io"
+    resources:
+      - kafkaprotocolfilters
+    verbs:
+      - get
+      - list
+      - watch
+      # - create
```

Comment:
we may as well delete the comment out lines.  The Operator is never going to be in the business of updating the Filter's spec.